test/controllers/api/v1/api_controller_test.rb

```diff
@@ -87,20 +87,133 @@ def setup
     assert_equal '{"test_key":"test_value"}', collector.payload
   end
 
-  test 'request validators with token should succeed' do
+  test 'GET api_v1_validators with token returns all data' do
+    validator = create(:validator, :with_score, account: 'Test Account')
```

Comment:
add more validators to see if it's all visible. Also make sure to check if mainnet validators are not shown if testnet is chosen